src/types.ts

```diff
@@ -3,7 +3,7 @@ import ts from 'typescript'
 export type LiteralType = string | number | boolean
 
 export type ZodToTsOptions = {
-	resolveNativeEnums?: boolean
```

Comment:
We still need to keep this option as it would be a breaking change to remove it. I just mean we should add a `/** @deprecated */` comment over it.

Reply:
Done